UI/src/app/pages/mapping/components/panel/panel-table/panel-table.component.ts

```diff
@@ -0,0 +1,67 @@
+
+import { Component, OnInit, Input, ViewChild, ElementRef, Injector, ChangeDetectionStrategy, ChangeDetectorRef } from '@angular/core';
+import { Store, select } from '@ngrx/store';
+import { Observable } from 'rxjs';
+import { CommentsService } from 'src/app/pages/mapping/services/comments.service';
+import { DragService } from 'src/app/pages/mapping/services/drag.service';
+import { Overlay, CdkOverlayOrigin, OverlayRef, OverlayConfig } from '@angular/cdk/overlay';
```

Comment:
try to follow the next order of importing:
1) foreign libs
2) your own scripts